config/internal/ml-metadata/metadata-envoy.deployment.yaml.tmpl

```diff
@@ -118,6 +119,7 @@ spec:
           volumeMounts:
             - mountPath: /etc/tls/private
               name: proxy-tls
+      {{ end }}
```

Comment:
(code cleanliness)  I think this is one indent off from its matching conditional above (6 vs 8 whitespaces